agot-bg-game-server/src/common/ingame-game-state/westeros-game-state/wildlings-attack-game-state/WildlingsAttackGameState.ts

```diff
@@ -107,9 +107,21 @@ export default class WildlingsAttackGameState extends GameState<WesterosGameStat
     }
 
     firstStart(wildlingStrength: number, participatingHouses: House[] = []): void {
+        if (this.game.wildlingStrength != wildlingStrength) {
+            this.game.wildlingStrength = wildlingStrength;
+            this.entireGame.broadcastToClients({
+                type: "change-wildling-strength",
+                wildlingStrength: wildlingStrength
+            });
+        }
+
         this.wildlingStrength = wildlingStrength;
         this.participatingHouses = participatingHouses;
 
+        this.biddingResults = null;
+        this._lowestBidder = null;
+        this._highestBidder = null;
```

Comment:
Why putting these initializations?

Reply:
Same reason as the other comment. I tried to solve the UX issue and thought maybe the old value is still hold here. But you are right, as we create always new instances of the states it isn't necessary. I remove it and try to solve it in the component.